frontend/src/routes/_dashboard/index.lazy.tsx

```diff
@@ -55,129 +59,123 @@ function AuthedView() {
+                  </MenubarContent>
+                </MenubarMenu>
+              </Menubar>
+            </div>
+          </div>
 
-      <Table>
-        <TableHeader className="bg-toggle">
-          <TableRow className="border-none">
-            <TableHead>Name</TableHead>
-            <TableHead className="flex items-center gap-2">
-              Last Updated
-              <ArrowDown size={15} />
-            </TableHead>
-            <TableHead>Status</TableHead>
-            <TableHead className="text-right">Actions</TableHead>
-          </TableRow>
-        </TableHeader>
-        <TableBody>
-          {projects.map((project) => (
-            <TableRow className="border-border cursor-pointer" key={project.id}>
-              <TableCell className="font-medium flex items-center gap-3">
-                <Folder size={18} />
-                {project.name}
-              </TableCell>
-              <TableCell>{project.updated_at}</TableCell>
-              <TableCell>
-                <StatusBadge color={getBadgeColor(project.tracker)}>
-                  {project.status}
-                </StatusBadge>
-              </TableCell>
-              <TableCell className="flex justify-end">
-                <div className="w-fit flex items-center gap-3">
-                  Settings
-                  <Settings width={18} />
-                </div>
-              </TableCell>
-            </TableRow>
-          ))}
-        </TableBody>
-      </Table>
+          <Table>
+            <TableHeader className="bg-toggle">
+              <TableRow className="border-none">
+                <TableHead>Name</TableHead>
+                <TableHead>Description</TableHead>
+                <TableHead className="flex items-center gap-2">
+                  Last Updated
+                  <ArrowDown size={15} />
+                </TableHead>
+                <TableHead>Status</TableHead>
+                <TableHead className="text-right">Actions</TableHead>
+              </TableRow>
+            </TableHeader>
+            <TableBody>
+              {projectList?.map((project: any) => (
+                <TableRow
+                  className="border-border cursor-pointer"
+                  key={project.id}
+                >
+                  <TableCell className="font-medium flex items-center gap-3">
+                    <Folder size={18} />
+                    {project.slug}
+                  </TableCell>
+                  <TableCell>{project.description}</TableCell>
+                  <TableCell>{formatedDate(project.updated_at)}</TableCell>
+                  <TableCell>
+                    <StatusBadge
+                      color={
+                        project.healthy_services === project.total_services
+                          ? "green"
+                          : project.healthy_services === 0 &&
+                              project.total_services > 0
+                            ? "red"
+                            : "yellow"
```

Comment:
```suggestion
                          : project.healthy_services === 0
                            ? "red"
                            : "yellow"
```
the other condition is unnecessary because in that case `total_services` will be always bigger than zero.